scripts/zones/Uleguerand_Range/npcs/Buffalostalker_Dodzbraz.lua

```diff
@@ -1,22 +1,30 @@
 -----------------------------------
 -- Area: Uleguerand Range
---  NPC: Buffalostalker Dodzbraz
+-- NPC: Buffalostalker Dodzbraz
 -- Type: Quest NPC
 -- !pos -380.171 -24.89 -180.797 5
+-- !additem 1667 (Cluster Core)
 -----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHas(trade, { { 1667, 2 } }) then -- Cluster Core x2
+    player:startEvent(8, 1667)
+    end
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(6)
+    player:startEvent(6, 1667)
 end
 
 entity.onEventUpdate = function(player, csid, option)
 end
 
 entity.onEventFinish = function(player, csid, option)
+    if csid == 8 then
+    npcUtil.giveItem(player, 5267)
```

Comment:
line 25+26 should be indented like line 17.

Reply:
Alright, I've done this and commit